app/models/cdo/create/select-address.js

```diff
@@ -10,7 +10,12 @@ function ViewModel (details, addresses = [], error) {
 
   this.model = {
     backLink: details?.source === 'create' ? routes.ownerDetails.get : details?.backLink,
-    changePostcodeLink: details?.source === 'create' ? routes.address.get : `${details?.backLink}#postcode`,
+    changeAddressLink: details?.source === 'create'
+      ? routes.address.get
+      : `${routes.editAddress.get}/${details.personReference}/true${details.srcHashParam}`,
```

Comment:
not so sure about /true...could it change /current?  query param would probably be better option...but understand it is likely more work.